client/webserver/site/src/js/markets.js

```diff
@@ -539,7 +559,7 @@ export default class MarketsPage extends BasePage {
     })
 
     page.lotSize.textContent = Doc.formatCoinValue(market.baseCfg.lotSize / 1e8)
-    page.rateStep.textContent = Doc.formatCoinValue(market.quoteCfg.rateStep / 1e8)
+    page.rateStep.textContent = parseFloat(Doc.formatCoinValue(market.quoteCfg.rateStep / 1e8))
```

Comment:
This appears to be the same as just assigning the value.

`page.rateStep.textContent = market.quoteCfg.rateStep / 1e8`